backend/db/models.py

```diff
@@ -130,6 +130,7 @@ class Adjudication(db.Model, BaseMixin):
     performance_id = db.Column(db.Integer, db.ForeignKey('performance.id'))
     nomination_comment = relationship('NominationComment', back_populates="adjudication")
 
+    #TODO: shouldn't this be in create_adjudications in service instead?
```

Comment:
It might be cleaner to throw some of this in a service, but I think it might also be okay to leave it here